docs/source/manual/configuration.rst

```diff
@@ -277,6 +277,8 @@ Name                     Default             Description
 ======================== ==================  ======================================================================================
 port                     8080                The TCP/IP port on which to listen for incoming connections.
 bindHost                 (none)              The hostname to bind to.
+inheritChannel           false               If your application launch by the `Server::Starter`_, please set ``true``.
```

Comment:
I would prefer the description to be a little more generic. What do you think about 
"Whether this connector uses a channel inherited from the JVM. Use it with `Server::Starter`, to launch an instance of Jetty on demand"? 

Also the property need to be set in the HTTP connector YAML configuration.